README.md

```diff
@@ -1,3 +1,134 @@
+automatically scheduling calendar events from tasks and finding the most efficient order of travel between different
+tasks. These features, ultimately, will empower the user to easily plan how they will take action.
+
+We are NOT aiming to simply embed multiple Google products onto a single page.
+Rather, we are aiming to present summary information from each service
+(i.e. You have 35 unread emails in the last 7 days vs. presenting a list of all unread emails
+[which would be redundant])
+
+We are NOT aiming to teach users who do not already use these features on a regular basis since this is targeted towards
+power users.
+
+We are NOT aiming to rewrite any existing functionality such as writing our own function to send an email,
+creating our own Google Tasks, or making our own calendar.
+
+We are NOT aiming to store any user information that might be displayed on our dashboard -
+we are simply extracting useful details from the information to digest and display back to the user.
+However, this could turn out to be a stretch goal if we wish to cache information to handle large amounts of data.
+
+
+## Get Started
+This project is configured to deploy on Google App Engine.
+
+### Prerequisites
+You must have Java 8 or above, Apache Maven, and the Google Cloud SDK installed and configured on your local machine.
+Here's how to install [Maven](https://maven.apache.org/install.html), [Java](https://java.com/en/download/help/download_options.xml)
+and [The Google Cloud SDK](https://cloud.google.com/sdk/install).
+
+### Google Cloud Platform
+1) To get started, begin by proceeding to [Google Cloud Console](https://console.cloud.google.com) and creating an account.
+
+2) Proceed to "Create an Empty Project". You can give the project any name you'd like.
+
+3) Once you've created your project, you should be provided with a "Project ID". Note that this is different than your
+project name, and should be in the following format: "sample-project-id".
+
+4) Clone this repository. In the [pom.xml](pom.xml) file, you should see the following code block:
+    ```$xslt
+        <groupId>com.google.sps</groupId>
+        <artifactId>INSERT_PROJECT_ID_HERE</artifactId>
+        <version>1</version>
+        <packaging>war</packaging>
+    ```
+Replace the `artifactId` with your project ID.
+
+5) In the [pom.xml](pom.xml) file, you should see the following code block:
+    ```$xslt
+    <plugin>
+        <groupId>com.google.cloud.tools</groupId>
+        <artifactId>appengine-maven-plugin</artifactId>
+        <version>2.2.0</version>
+    </plugin>
+    ```
+   Replace it with this:
+   ```$xslt
+   <plugin>
+       <groupId>com.google.cloud.tools</groupId>
+       <artifactId>appengine-maven-plugin</artifactId>
+       <configuration>
+           <deploy.version>1</deploy.version>
+           <deploy.projectId>YOUR_PROJECT_ID</deploy.projectId>
+       </configuration>
+       <version>2.2.0</version>
+   </plugin>
+   ```
```

Comment:
Maybe mention that `-Dapp.deploy.version=<VERSION>` and `-Dapp.deploy.projectId=<PROJECT_ID>` are also options? (Especially because that's what cloudbuild.yaml does)

Reply:
Cool idea - thanks!